vector/src/main/java/fr/gouv/tchap/features/login/TchapLoginFragment.kt

```diff
@@ -93,7 +92,7 @@ class TchapLoginFragment @Inject constructor() : AbstractLoginFragment<FragmentT
     private fun submit() {
         cleanupUi()
 
-        login = views.tchapLoginField.text.toString()
+        val login = views.tchapLoginField.text.toString()
```

Comment:
this will just postpone the crash to the next line, you have to do the same for the password field